proto/substrait/plan.proto

```diff
@@ -50,6 +51,11 @@ message Plan {
   // An optional list of bindings for dynamic parameters used in this plan.
   // Each binding maps a parameter_anchor to its corresponding runtime value.
   repeated DynamicParameterBinding parameter_bindings = 7;
+
+  // An optional list of concrete types (i.e., no alias!). Parameterized types
```

Comment:
Maybe reword the comment for clarity, it says "no alias!" but the field is a `repeated TypeAlias`.